fetch.bs

```diff
@@ -3848,11 +3848,18 @@ steps:
 <ol>
  <li><p>Let <var>request</var> be <var>fetchParams</var>'s <a for="fetch params">request</a>.
 
- <li><p>Let <var>response</var> be null.
+ <li>
+  <p>Let <var>response</var> be the result of invoking <span>consume a preloaded resource</span>
+  for <var>request</var>.
+
+  <p class="note">This is done in main fetch and not in HTTP-fetch to avoid blocking or race
+  conditions between the networking queue and the document queue. The implication is that if a
+  fetch redirects to a URL that was preloaded, it would not be consumed.</p>
```

Comment:
I don't understand this comment. HTTP-redirect fetch invokes main fetch at the end.

Reply:
Hmm looking at this again, this patch is wrong. Will resubmit